docs/frontend.md

```diff
@@ -132,7 +116,19 @@ workspace packages. Otherwise, it can inflate the size of the compiled bundles a
 performance impact on users. Similarly, consider using a tool like [`yarn-deduplicate`](https://github.com/scinos/yarn-deduplicate)
 to deduplicate resolved package versions within the Yarn lockfile.
 
-### Components
+### Localization
+
+See [`@18f/identity-i18n` package documentation](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/i18n/README.md).
```

Comment:
I think we could do a relative link here on GH?
```suggestion
See [`@18f/identity-i18n` package documentation](../app/javascript/packages/i18n/README.md).
```

Reply:
Yeah, that's true. I think I tend to shy away from relative links in GitHub documents since they don't always work when the document is presented elsewhere, like READMEs on NPM packages. But I don't think that's something we need to worry about with this document.